packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -86,7 +86,12 @@ class OAuthController {
 
         try {
             logCtx = await logContextGetter.create(
-                { id: String(activityLogId), operation: { type: 'auth', action: 'create_connection' }, message: 'Authorization OAuth' },
+                {
+                    id: String(activityLogId),
+                    operation: { type: 'auth', action: 'create_connection' },
+                    message: 'Authorization OAuth',
+                    expiresAt: new Date(Date.now() + 300 * 1000).toISOString()
```

Comment:
Feel like this expiresAt should be a constant somewhere since it is repeated in so many places

```
new Date(Date.now() + 300 * 1000).toISOString()
```